app/controllers/recipe_foods_controller.rb

```diff
@@ -0,0 +1,62 @@
+class RecipeFoodsController < ApplicationController
+  before_action :authenticate_user!
+
+  def new
+    @recipe = Recipe.find(params[:recipe_id])
+    used = @recipe.food_used
+    @foods = Food.where(user_id: current_user.id).reject { |food| used.include?(food.name) }
```

Comment:
**Optional Change**
- [ ] It is good to keep the controller as lean as possible. This line could be a method in the Recipe Food model and just called in the controller.